code/datums/jobs/jobs_civilian.dm

```diff
@@ -52,6 +52,7 @@ ABSTRACT_TYPE(/datum/job/civilian)
 	slot_foot = list(/obj/item/clothing/shoes/brown)
 	slot_glov = list(/obj/item/clothing/gloves/black)
 	slot_poc1 = list(/obj/item/paper/botany_guide)
+	slot_poc2 = list(/obj/item/plantanalyzer)
```

Comment:
If you're spawning these in botanist's pockets now then the stock in the vendor should be reduced.